test/es6/lambda1.js

```diff
@@ -270,6 +270,52 @@ var tests = [
         }
     },
     {
+        name: "eval('arguments') works correctly within lambdas",
+        body: function () {
+            var arguments = 'not arguments object';
+            assert.areEqual(arguments, (() => eval("arguments"))(), "arguments in lambda should bind to outside arguments since it is not given its own arguments binding");

```

Comment:
Both should refer to the variable ('not an arguments object'). The `var arguments` line shadows the default arguments object, so both the first param to `areEqual` and the `arguments` inside the lambda should refer to that.